src/lib/variants.ts

```diff
@@ -327,6 +318,17 @@ export const isNotExpired = (now: Date = new Date()): Filter => ({
     },
 });
 
+export const optOutOfArticleCount: Filter = {
+    id: 'optOutOfArticleCount',
+    test: (test, targeting) => {
+        if (test.articlesViewedSettings) {
+            return !targeting.hasOptedOutOfArticleCount;
```

Comment:
Should this have the `!` ?

Reply:
If so, then I think the filter should be renamed to `shouldShowArticleCountOptOutWidget` or something like that

Reply:
The filter is to prevent users from being put into a test which has article count, if they have opted out

Reply:
So the filter will return `false` (meaning do not put them in the test) if the test has article count and the user has opted out

Reply:
Yeah I wasn't 100% on our naming convention for filters. But yeah, filter returns true if it's okay for the user to see the epic

Reply:
Yes the naming is a bit confusing with these filters! A more descriptive name might be something like `respectArticleCountOptOut`

Reply:
Yeah totally agree - happy to go ahead and make that change